sdk/attestation/azure-security-attestation/azure/security/attestation/aio/__init__.py

```diff
@@ -2,9 +2,21 @@
 # --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from ._attestation_client import AttestationClient
-__all__ = ['AttestationClient']
+from ._client_async import AttestationClient
+#from .._administration_client import AttestationAdministrationClient
```

Comment:
Did this file get removed? If so, you can remove and why was it removed? Otherwise we should include

Reply:
The aio is a bare-bones implementation which only does one API. Completing the async API is on my long list of things to do.